docs/source/mypy_daemon.rst

```diff
@@ -94,6 +94,118 @@ Use ``dmypy --help`` for help on additional commands and command-line
 options not discussed here, and ``dmypy <command> --help`` for help on
 command-specific options.
 
+Additional daemon flags
+***********************
+
+.. option:: --status-file FILE
+
+   Status file to retrieve daemon details. This is normally a JSON file
+   that contains information about daemon process and connection. Default is
+   ``.dmypy.json``.
+
+.. option:: --log-file FILE
+
+   Direct daemon stdout/stderr to ``FILE``. This is useful for debugging daemon
+   crashes, since the server traceback may be not printed to client stderr. Only
+   available for ``start``, ``restart``, and ``run`` commands.
+
+.. option:: --timeout TIMEOUT
+
+   Server shutdown timeout (in seconds). Only available for ``start``,
+   ``restart``, and ``run`` commands.
+
+.. option:: --fswatcher-dump-file FILE
+
+   Collect information about the current file state. Only available for
+   ``status`` command.
+
+.. option:: --perf-stats-file FILE
+
+   Write performance telemetry information to the given file. Only available
+   for ``check``, ``recheck``, and ``run`` commands.
+
+.. option:: --update FILE
+
+   Files in the run to add or check again, may be repeated. Default: all
+   files from the previous run. Only available for ``recheck`` command.
+
+.. option:: --remove FILE
+
+   Files to remove from the run, may be repeated. Only available for
```

Comment:
This suggests that this is needed whenever files are removed, which is not the case? Similar to above, document that this can be used as an optimization to avoid looking at all source files for deletions.